lib/oauth.ts

```diff
@@ -36,6 +36,19 @@ export interface ProvidersValues {
   scopes: string[];
 }
 
+export interface AttachRequest {
+  provider: string;
+
+  // Exactly one of these user-selection fields must be provided.
```

Comment:
Does this comment appear in-IDE as a JSDOC? I think you need to use `/** */` comment syntax for that

Reply:
Does `///` doc comment syntax work as well?